cpp/src/io/orc/aggregate_orc_metadata.hpp

```diff
@@ -115,11 +113,11 @@ class aggregate_orc_metadata {
    *
    * Stripes are potentially selected from multiple files.
```

Comment:
this would really benefit from a `@return` description

Reply:
Oh no there are many functions need to be updated with `@param` and `@return`.